project/plugins.sbt

```diff
@@ -11,8 +11,6 @@ addSbtPlugin("org.scala-native" % "sbt-scala-native" % "0.3.7")
 
 //addSbtPlugin("com.lihaoyi" % "workbench" % "0.4.1")
 
-addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.1.1")
```

Comment:
Why remove this? SBT still needs to sign jars on publish, right?

Reply:
sbt-CI-release takes care of signing

Reply:
ok thx

Reply:
It will require somebody to setup the pgp keys and sonatype credentials as indicated on 
https://github.com/olafurpg/sbt-ci-release#travis